ProductionModePlugin.js

```diff
@@ -65,6 +73,7 @@ ProductionModePlugin.prototype.apply = function(compiler) {
     var request = this.state.current.request;
     if(param.isString() && param.string === "globalize" &&
           !util.isGlobalizeModule(request) &&
+          !isTransformExcluded(request) &&
           !(globalizeCompilerHelper.isCompiledDataModule(request))) {
```

Comment:
Can we simplify this and do something like the below (where returning `true` means process it, `false` skip it)?

```diff
-          !util.isGlobalizeModule(request) &&
-          !isTransformExcluded(request) &&
-          !(globalizeCompilerHelper.isCompiledDataModule(request))) {
+         moduleFilter(request))) {
```